### DISCUSSION TITLE
Does 'obsidian-git' honor .gitignore fully?

### DISCUSSION THREAD

**Discussion by RedwanFox** [Vinzent03/obsidian-git#677] [Problems] [answered]
I have added `.obsidian/workspace.json` to .gitignore because there lots of merge conflicts due different opened tabs in different obsidian instances. But I still get merge conflicts.

**Comment by Vinzent03** [comment, marked as answer]
Yes it does, but when you want to ignore a file that's already committed, you have to manually remove the file from the index with `git rm --cached <file>`
